subsys/usb/device_next/class/bt_hci.c

```diff
@@ -218,6 +325,7 @@ static void bt_hci_tx_thread(void *p1, void *p2, void *p3)
 			ep = bt_hci_get_int_in(c_data);
 			break;
 		case BT_HCI_H4_ACL:
+		case BT_HCI_H4_ISO:
 			ep = bt_hci_get_bulk_in(c_data);
```

Comment:
So far I couldn't make Linux host recognize ISO packets. It might be the case it ignores CIS creation commands if passed via userchan. If that's the case, userchan driver in Zephyr might have to do this instead